### PR TITLE
Fix data virt images and references

### LINKED ISSUE #148: [1.7.7] Use fuse7-tech-preview instead of fuse-7-tech-preview
A dash changes everything

### LINKED ISSUE #149: [1.7.7] Mismatch between imagestreams in fuse-online-install and operator template
fuse-online-install specifies the image stream for data virt as `fuse-ignite-komodo` and the tag as `1.7`:

https://github.com/syndesisio/fuse-online-install/blob/5032f8374613eeaea0a15e817006efdf2700c8a6/resources/fuse-online-image-streams.yml#L126-L143

however the internal template in the operator expects:

```
from:
          kind: ImageStreamTag
          name: fuse-komodo-server:latest
```

There are two issues logged for that: https://issues.jboss.org/browse/ENTESB-11036 https://issues.jboss.org/browse/ENTESB-11037

So either those two will be scheduled for CR2, or we need to edit the resources in fuse-online-install.

### LINKED ISSUE #150: [1.7.7] No switch for datavirt install
install_ocp.sh doesn't have a switch for installing with data virtualization

### PR DESCRIPTION
**Pull request by nicolaferraro**
This..
fix #148
fix #149
fix #150

To test it there are some manual steps to do, because it requires the following issues to be fixed:
- https://issues.jboss.org/browse/ENTESB-11037
- https://issues.jboss.org/projects/ENTESB/issues/ENTESB-11043
- https://github.com/syndesisio/syndesis/issues/6111 (fixed but not yet in latest release)

This one can be closed imo:
- https://issues.jboss.org/browse/ENTESB-11036

cc: @heiko-braun , @avano

### CODE REVIEW

File: install_ocp.sh [hunk 1]
```diff
@@ -580,6 +581,17 @@ EOT
         syndesis="${syndesis}${extra}"
     fi
 
+    local datavirt=$(hasflag --datavirt)
+    if [ -n "${datavirt}" ]; then
+        extra=$(cat <<EOT
+
+  addons:
```
Comment (heiko-braun) [inline install_ocp.sh]: Does the operator change config map and update the replica count when this is set?

Comment (nicolaferraro) [reply install_ocp.sh]: It does at install time. So if the `Syndesis` resource contains this config before the install starts, then Komodo is installed and the configuration is set accordingly.

But changing it at runtime should not have effects currently.

Comment (nicolaferraro) [reply install_ocp.sh]: I've verified it sets replicas to 1 and changes the syndesis-ui configmap.

Comment (heiko-braun) [reply install_ocp.sh]: Excellent! Good work!

**Pull request merged**